<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1473400641706" stop="1473400642144" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Suite : Test</name>
    <title>Suite : Test</title>
    <test-cases>
        <test-case start="1473400641785" stop="1473400641878" status="passed">
            <name>testcase1</name>
            <description type="text">This is the first test case</description>
            <steps>
                <step start="1473400641863" stop="1473400641878" status="passed">
                    <name>testcase1</name>
                    <title>This is the first test step for the first test case</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="critical"/>
                <label name="feature" value="New feature"/>
                <label name="host" value="XE-T-NITINGO"/>
                <label name="thread" value="5460@XE-T-NITINGO.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473400641878" stop="1473400641910" status="failed">
            <name>testcase2</name>
            <description type="text">This is the second test case</description>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at Newpackage.NewTest.testcase2(NewTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps>
                <step start="1473400641878" stop="1473400641894" status="failed">
                    <name>testcase2</name>
                    <title>This is the second test step for the first test case</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="normal"/>
                <label name="feature" value="Allure test report check"/>
                <label name="host" value="XE-T-NITINGO"/>
                <label name="thread" value="5460@XE-T-NITINGO.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473400641910" stop="1473400642097" status="passed">
            <name>testcase3</name>
            <description type="text">This is the third test case</description>
            <steps>
                <step start="1473400641910" stop="1473400642097" status="passed">
                    <name>testcase3</name>
                    <title>This is the third test step for the third test case</title>
                    <attachments>
                        <attachment title="testlogfile" source="86f0283d-6ada-4929-8e45-d4436c78c199-attachment.txt" type="text/plain"/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="trivial"/>
                <label name="feature" value="Allure test report check"/>
                <label name="host" value="XE-T-NITINGO"/>
                <label name="thread" value="5460@XE-T-NITINGO.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
